tezos/timing/src/lib.rs

```diff
@@ -912,6 +921,9 @@ mod tests {
         let block_hash = BlockHash::try_from_bytes(&vec![1; 32]).unwrap();
         let context_hash = ContextHash::try_from_bytes(&vec![2; 32]).unwrap();
 
+        TIMING_CHANNEL
```

Comment:
What happens when the other messages are received but the `InitTiming` one wasn't? Can it be made so that everything is ignored until this message is received?